crates/oxc_linter/src/rules/typescript/prefer_namespace_keyword.rs

```diff
@@ -88,18 +105,32 @@ fn test() {
 
     let fix = vec![
         ("module foo {}", "namespace foo {}", None),
-        ("declare module foo {}", "declare namespace foo {}", None),
+        ("module A.B {}", "namespace A.B {}", None),
         (
             "
-			declare module foo {
-			  declare module bar {}
+            module A {
+              module B {}
 			}
```

Comment:
Your string formatting is still wrong, I guess you are using tabs over spaces - which is something I do as well - If you are using Vim write `:retab` to convert all of them to spaces.
On VSCode there is a built-in command called `Convert Indentations to Spaces` which you can use.